File: test/resources/HelloProphecy/prophecy/pipelines/gold_sales/code/gold_sales/graph/Cleanup.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from gold_sales.config.ConfigStore import *
from gold_sales.udfs.UDFs import *

def Cleanup(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(
        datediff(col("account_open_date"), col("order_date")).alias("account_age_at_order_date"), 
        col("order_id"), 
        col("customer_id"), 
        col("amount"), 
        col("order_date"), 
        col("zipcode"), 
        col("account_flags")
    )

File: test/resources/HelloProphecy/prophecy_scala/pipelines/gold_sales/code/src/main/scala/io/prophecy/pipelines/gold_sales/graph/Cleanup.scala
package io.prophecy.pipelines.gold_sales.graph

import io.prophecy.libs._
import io.prophecy.pipelines.gold_sales.udfs.PipelineInitCode._
import io.prophecy.pipelines.gold_sales.udfs.UDFs._
import io.prophecy.pipelines.gold_sales.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Cleanup {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(
      datediff(col("account_open_date"), col("order_date"))
        .as("account_age_at_order_date"),
      col("order_id"),
      col("customer_id"),
      col("amount"),
      col("order_date"),
      col("zipcode"),
      col("account_flags")
    )

}
